tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php

```diff
@@ -260,6 +260,41 @@ public function testPostNewOkUrlExistWithAccent()
         $this->assertContains('/view/', $client->getResponse()->getTargetUrl());
     }
 
+    public function testPostNewOkUrlExistWithRedirection()
+    {
+        $this->logInAs('admin');
+        $client = $this->getClient();
+
+        $url = 'https://fashionunited.com/news/business/jigsaw-posts-8-percent-increase-in-annual-sales/2017070316404';
```

Comment:
Better use a bit.ly url or a t.co so it'll be obvious that the url will be redirected and given_url won't be the same as url.